### ISSUE TITLE
Background and foreground colors have a sufficient contrast ratio error axe-core Error:Cannot read property 'cells' of undefined for LH 7.0.1

### ISSUE THREAD

**Issue by bhaskar1603** [GoogleChrome/lighthouse#12043]
Provide the steps to reproduce
Run LH on https://gurugramproperty.in/
What is the current behavior?
shows red Question mark on accessibility test for both mobile and desktop report
Background and foreground colors have a sufficient contrast ratio error axe-core Error (Cannot read property 'cells' of undefined)

What is the expected behavior?
Shall show the accessibility score .

Environment Information
Lighthouse version: 7.0.1
Device: Emulated Desktop
Network throttling: 40 ms TCP RTT, 10,240 Kbps throughput (Simulated)
CPU throttling: 1x slowdown (Simulated)
Channel: devtools

User agent (host)
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4407.0 Safari/537.36

User agent (network)
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4143.7 Safari/537.36 Chrome-Lighthouse

CPU/Memory Power
1334

Axe version
4.1.1
